README.md

```diff
@@ -51,55 +50,41 @@ This procedure will end installing the following files:
     ${prefix}/share/applications/qsynth.desktop
     ${prefix}/share/locale/qsynth_*.qm
 
-Just launch ${prefix}/bin/qsynth and you're off (hopefully).
-
-Note that the default installation path ${prefix} is /usr/local.
+Now just launch ${prefix}/bin/qsynth and you're off (hopefully).
 
-If you're checking out from Git, you'll have to prepare the
-configure script just before you proceed with the above instructions:
+(Note that the default installation path ${prefix} is /usr/local.)
 
-   ./autogen.sh
 
-
-Configuration
--------------
+## Configuration
 
 Qsynth holds its settings and configuration state per user, in a file
 located as $HOME/.config/rncbc.org/Qsynth.conf . Normally, there's no
 need to edit this file, as it is recreated and rewritten everytime
 qsynth is run.
 
 
-Bugs
-----
+## Bugs
 
 Plenty still although this is beta software ;)
 
 
-Support
--------
+## Support
 
 Qsynth is open source free software. For bug reports, feature requests,
-discussion forums, mailling lists, or any other matter related to the
-development of this piece of software, please use the Sourceforge
-project page (http://sourceforge.net/projects/qsynth).
+discussion forums, or any other matter related to the
+development of this piece of software, please use the [Github page](https://github.com/rncbc/qsynth).
```

Comment:
I changed this since it seems like you are using GitHub a lot and since it is easier to use for new contributors versus Sourceforge. Will change back if you say so...

Reply:
Okay, I will change that back then. Do you still accept PRs coming through github?

Reply:
This should now be fixed :+1: